site/package.json

```diff
@@ -1,9 +1,7 @@
 {
-  "name": "grammy-website",
-  "version": "1.0.0",
-  "description": "Main documentation site for grammY",
-  "main": "index.js",
   "private": true,
+  "license": "MIT",
+  "author": "KnorpelSenf",
```

Comment:
You can remove this, it's a bit tilted anyway, I'm by far not the only author here